toolbar-extender/index.ts

```diff
@@ -3,6 +3,9 @@ import { CommonModule } from "@angular/common";
 import { EXTERNAL_TOOLBAR_ITEMS_PROVIDER } from "./toolbar-items-provider";
 import { VIDEO_TOOLBAR_ITEM_PROVIDER } from "../sitefinity-videos/sitefinity-videos-toolbar-item-provider";
```

Comment:
why we use one index for two extensions, it may confuse devs if they wish to let say remove the whole ```/toolbar-extender``` folder they will loose the video extension as well

Reply:
i will move both of them under toolbar-extender